### PR TITLE
Release 2.9.7 beta2

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming second beta of the v2.9.7 series? Well, you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
- Notifications beta enabled 
image
- [x] ~~If there are any new metrics, ensure that central and desktop.github.com have been updated~~

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.9.7-beta2": [
+      "[New] Initial support for system notifications when checks fail in Windows - #13655",
+      "[Improved] Upgrade to Electron v16.0.8 - #13814",
+      "[Improved] Use perl to resolve symlinks on macOS - #13764. Thanks @spotlightishere!"
```
Comment (sergiou87) [inline changelog.json]: I think I'd be more generic with this:
```suggestion
      "[Fixed] Fix CLI tool on macOS Monterey - #13764. Thanks @spotlightishere!"
```

**Review by sergiou87 — CHANGES_REQUESTED** [review]

File: changelog.json (hunk 1 above)
Comment (sergiou87) [reply changelog.json]: You might need to move lines around (Fixed lines are above Improved)

**Review by billygriffin — APPROVED** [review]
🚀 1️⃣ 6️⃣

**Review by sergiou87 — APPROVED** [review]
🚀 🐸 🕴️

**Pull request merged**